themes/default/content/blog/property-search/index.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Property Search: Enhanced Search Capabilities on Pulumi Cloud"
+allow_long_title: true
+# The date represents the post's publish date,
+# and by default corresponds with the date this file was generated.
+# Posts with future dates are visible in development,
+# but excluded from production builds.
+# Use the time and timezone-offset portions of of this value
+# to schedule posts for publishing later.
+date: 2023-06-13T09:00:08-07:00
+
+# Use the meta_desc property to provide a brief summary
+# (one or two sentences) of the content of the post,
+# which is useful for targeting search results or social-media previews.
+# This field is required or the build will fail the linter test.
+# Max length is 160 characters.
+meta_desc: Pulumi Cloud launches Property Search, a feature on top of Resource Search that opens search to resource properties.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect ratio
+# has been provided for you.
+meta_image: meta.png
+
+# At least one author is required.
+# The values in this list correspond with the `id` properties
+# of the team member files at /data/team/team.
+# Create a file for yourself if you don't already have one.
+authors:
+    - meagan-cojocar
+    - bryce-lampe
+
+# At least one tag is required.
+# Lowercase, hyphen-delimited is recommended.
+tags:
+    - pulumi-cloud
+    - features
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details,
+# and please remove these comments before submitting for review.
+---
+
+When we launched [Resource Search for Pulumi Cloud](/blog/pulumi-insights) in April we extended the capability of Pulumi Cloud beyond just managing infrastructure as code to managing your cloud footprint. Today, we're excited to announce that we are taking this a step further and exposing a much larger set of data to search over: resource properties. Pulumi Property Search enables you to search on a wide array of resource properties, everything from instance type, to database version, to region and availability zone, to whether resources are open to the internet.
+
+<!--more-->
+
+Property Search is applicable across all cloud resources, whether they reside in AWS, Google Cloud, Azure, or any of Pulumi's 100+ cloud providers. Pulumi Cloud Resource Search is available on the Enterprise and Business Critical editions of Pulumi Cloud.
```

Comment:
Was the second sentence not correct?  Or perhaps I was reading it wrong?

```suggestion
Pulumi Resource Search, including the ability to search on property data, is applicable across all cloud resources, whether they reside in AWS, Google Cloud, Azure, or any of Pulumi's 100+ cloud providers. Support for querying on property data is available in the Enterprise and Business Critical editions of Pulumi Cloud.
```

Also, should we link off to pricing page for edition information?

Reply:
> Was the second sentence not correct? Or perhaps I was reading it wrong?

It was incorrect! Fixed in my latest